src/unity/python/turicreate/test/test_sentence_classifier.py

```diff
@@ -85,6 +88,25 @@ def test_evaluate(self):
         """
         self.model.evaluate(self.docs)
 
+    def test_export_coreml(self):
+        filename = tempfile.mkstemp('bingo.mlmodel')[1]
+        self.model.export_coreml(filename)
+
+    @unittest.skipIf(sys.platform != 'darwin', 'Only supported on Mac')
```

Comment:
This isn't quite the right check (since predictions are only supported on macOS 10.13+).

You may want to put your Core ML export test in https://github.com/apple/turicreate/blob/master/src/unity/python/turicreate/test/test_coreml_export.py to share the common code there for evaluating turicreate predictions against Core ML. If you'd prefer to keep it here, the skipIf needs to check macOS version.

Reply:
I see there is an `if` statement within for the macOS version, but I think on prior macOS versions we'd want this test to report as `skipped` rather than `passed`.